app/assets/javascripts/assignments_file_upload.js

```diff
@@ -0,0 +1,141 @@
+/*
+This file is part of SSID.
+
+SSID is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+SSID is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with SSID.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+var checkbox = document.getElementById('assignment_mapbox');
+var details_div = document.getElementById('map_file_details');
+checkbox.onchange = function () {
```

Comment:
The 3-dot menu site-wide is not working e.g. cannot access Edit/Delete a course ...